CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 ## 0.10.6
```

Comment:
need to add a new version for alchemy exp 1.0.0 + its new change log.

Reply:
1.0.0-exp-rc.1? Making it clear that this is still a release candidate and not official 1.0.0?